crates/types/src/certificate.rs

```diff
@@ -36,7 +36,7 @@ pub struct DACertificate<TYPES: NodeType> {
     pub view_number: TYPES::Time,
 
     /// committment to the block
-    pub block_commitment: Commitment<TYPES::BlockType>,
+    pub block_commitment: Commitment<TYPES::BlockPayload>,
```

Comment:
I think we should be careful to refer to objects of this type, wherever we have them, as `payload_commitment` or something, since `block_commitment` could also mean the commitment to the entire block, including header